tests/conftest.py

```diff
@@ -305,6 +305,7 @@ def kill(self):
             """Clean up all built VMs"""
             for vm in self.vms:
                 vm.kill()
+                vm.jailer.cleanup()
```

Comment:
mh, how will this show up in the pytest failures report though? Right now there's multiple asserts in the kill/cleanups code

Reply:
The test may pass, but the fixture could ERROR.

Reply:
Ah, right, sorry. I saw "conftest.py" and thought they'd get cleanup at teardown of the pytest runtime, not at teardown of each individual test. Its currently a fixture ERROR too with the destructor solution, so nothing changes then :D